packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -465,31 +485,44 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
         'data-is-focusable': false
       }
     );
+    const ariaDescribedBy = buttonProps.ariaDescription || '';
+
+    if (keytipProps && menuProps) {
+      keytipProps = {
+        ...keytipProps,
+        hasMenu: true
+      };
+    }
 
     return (
-      <div
-        role={ 'button' }
-        aria-labelledby={ buttonProps.ariaLabel }
-        aria-disabled={ disabled }
-        aria-haspopup={ true }
-        aria-expanded={ this._isExpanded }
-        aria-pressed={ this.props.checked }
-        aria-describedby={ buttonProps.ariaDescription }
-        className={ classNames && classNames.splitButtonContainer }
-        onKeyDown={ this._onSplitButtonContainerKeyDown }
-        ref={ this._splitButtonContainer }
-        data-is-focusable={ true }
-        onClick={ !disabled && !primaryDisabled ? this._onSplitButtonPrimaryClick : undefined }
-        tabIndex={ !disabled ? 0 : undefined }
-      >
-        <span
-          style={ { 'display': 'flex' } }
-        >
-          { this._onRenderContent(tag, buttonProps) }
-          { this._onRenderSplitButtonMenuButton(classNames) }
-          { this._onRenderSplitButtonDivider(classNames) }
-        </span>
-      </div>
+      <KeytipData keytipProps={ keytipProps } disabled={ disabled }>
+        { (keytipAttributes: any): JSX.Element => (
+          <div
+            data-ktp-target={ keytipAttributes['data-ktp-target'] }
+            role={ 'button' }
+            aria-labelledby={ buttonProps.ariaLabel }
+            aria-disabled={ disabled }
+            aria-haspopup={ true }
+            aria-expanded={ this._isExpanded }
+            aria-pressed={ this.props.checked }
+            aria-describedby={ ariaDescribedBy + (keytipAttributes['aria-describedby'] || '') }
```

Comment:
are you guaranteed that ariaDescribedBy and the aria-describedby from keytipAttributes will have a space in between them? Consider something like `${ariaDescribedby} ${keytipAttributes['ariadescribedby']}`

Reply:
The beginning space is added in the getAriaDescribedBy function